client/src/pages/PostSolution.tsx

```diff
@@ -44,23 +46,24 @@ const PostSolution = () => {
   const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
     e.preventDefault();
     // TODO: Form validation
-    console.log(formData);
-    console.log(challengeId);
+    const data = new FormData();
```

Comment:
What is this form data used for?

Reply:
formData is the state for PostSolution page, it contains the description, title and the file object. FormData is the browser api for multipart file uploads, which we now use.

Reply:
Refactored to cuz the names were too similar. formData is now postSolutionState